keyboards/inett_studio/sq80/keymaps/ANSI/keymap.c

```diff
@@ -1,13 +1,14 @@
-#include QMK_KEYBOARD_H
+
```

Comment:
What I meant before was this entire file should be deleted with:

```
git rm -- keyboards/inett_studio/sq80/keymaps/ANSI/keymap.c
```

You don't need it, because it's the same as `default`.